src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -938,6 +938,18 @@ protected ContestInformation getContestInformation(IInternalContest contest){
 
     private void setScoreboardFreezeTime(IInternalContest contest, String scoreboardFreezeTime) {
         ContestInformation contestInformation = contest.getContestInformation();
+        // adapted from core.util.JSONTool
+        // seems the yaml is converting the 1:00:00 into 3600
+        if (scoreboardFreezeTime.length() > 2) {
+            if (!scoreboardFreezeTime.contains(":")) {
+                try {
+                    long seconds = Long.parseLong(scoreboardFreezeTime);
+                    scoreboardFreezeTime = ContestTime.formatTime(seconds);
+                } catch (NumberFormatException e) {
+                    System.out.println("attempting to parse " + scoreboardFreezeTime + " failed with " + e.getMessage());
```

Comment:
This will not stop approval of this PR.
Note syntaxError method was not used to throw the exception.
Should use syntaxError for a couple of reasons:

1 - syntaxError should be used because if in the future we
refactor how to throw or log the exception there would
be only one place to make that change.
2 - syntaxError is used consistently throughout the class,
for error handling we should be consistent with the 
previous code to make the code easier to read and debug.

Reply:
changed to use syntaxError()